worker/ffmpeg_worker.py

```diff
@@ -0,0 +1,124 @@
+
+import os
+import subprocess
+from typing import Tuple
+
+from flask import Flask
+from flask import request
+from flask import safe_join
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+app = Flask(__name__)
+
+
+@app.errorhandler(NotFound)
+def _handle_gcloud_not_found(_):
+    return 'Google Cloud storage object not found', 404
+
+
+@app.route('/ffmpeg', methods=['POST'])
+def process_ffmpeg_command():
+    """Runs ffmpeg according to the request's specification.
+
+    The POST request supports the following arguments:
+        input_file: the input file in the Google Cloud Bucket specified
+                    Example: bucket_name/path/to/file
+        output_file: the output file that should be stored in the
+                     Google Cloud Bucket specified
+                     Example: bucket_name/path/to/file
+    This route changes the video container format by running the following:
+        ffmpeg -i {input_file} {output_file}
+
+    Args:
+
+    Returns:
+        A string with the ffmpeg logs.
+    """
+    remote_input_file = request.form['input_file']
+    local_input_file = _localize_filename(remote_input_file)
+    download_file(local_input_file, remote_input_file)
+    remote_output_file = request.form['output_file']
+    local_output_file = _localize_filename(remote_output_file)
+
+    ffmpeg_logs = subprocess.run(
+        ["ffmpeg", "-i", local_input_file, local_output_file],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+        check=False).stdout
+    os.remove(local_input_file)
+    upload_file(local_output_file, remote_output_file)
+    os.remove(local_output_file)
+    return ffmpeg_logs
+
+
+def download_file(local_filename: str, remote_filename: str) -> None:
+    """Downloads the file from Google Cloud Storage
+
+    This function downloads the file to the /tmp directory.
+
+    Args:
+        local_filename: the filepath of the downloaded file
+        remote_filename: the filename preceded by the Google Cloud Bucket name
+                         Example: bucket_name/path/to/file
+
+    Returns:
+        None
+    """
+    client = storage.Client()
+    local_filename = _localize_filename(remote_filename)
+    os.makedirs(os.path.dirname(local_filename), exist_ok=True)
+    with open(local_filename, 'wb') as downloaded_file:
+        client.download_blob_to_file('gs://' + remote_filename, downloaded_file)
+
+
+def upload_file(local_filename: str, remote_filename: str) -> None:
+    """Uploads the file from Google Cloud Storage.
+
+    This function uploads the file to the Google Cloud Bucket specified.
+
+    Args:
+        local_filename: the filepath of the local file to upload
+        remote_filename: the filename preceded by the Google Cloud Bucket name
+                         Example: bucket_name/path/to/file
+
+    Returns:
+        None
+    """
+    client = storage.Client()
+    bucket_name, blob_name = _split_remote_filename(remote_filename)
```

Comment:
no need for a separate function. just do 
bucket_name, blob_name =  os.path.normpath(remote_filename).split('/', 1) 
and let it throw an exception if there  is not "/" in the filename  .